src/transport/PeerConnectionState.h

```diff
@@ -62,6 +64,9 @@ class PeerConnectionState
     void SetTransport(Transport::Base * transport) { mTransport = transport; }
     Transport::Base * GetTransport() { return mTransport; }
 
+    bool isPeerMsgCounterSynced() { return (mPeerMessageIndex != kUndefinedMessageIndex); }
```

Comment:
I don't think `kUndefinedMessageIndex` (which is `UINT32_MAX`) is reserved in the spec. it should be a valid index, check it against `mPeerMessageIndex` is not safe.

Reply:
Acked.